media-fonts/inconsolata-lgc/inconsolata-lgc-1.1.0_p20140917.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+
+DESCRIPTION="Modified version of Inconsolata Hellenic adding the Cyrillic alphabet"
+HOMEPAGE="https://github.com/DeLaGuardo/Inconsolata-LGC"
+SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/tarball/d086a618248121d61e3f3de64b4301230d1c860c -> ${P}.tar.gz"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
+
+DEPEND=""
+
+S="${WORKDIR}/DeLaGuardo-Inconsolata-LGC-d086a61"
```

Comment:
You could cut it off the hash-var, e.g. `${EGIT_COMMIT::7}` (if I'm counting right).

Reply:
Okay, I'll push a revision to the PR soon for this and the other suggestion above.